assets/javascripts/modules/accordion.js

```diff
@@ -83,6 +83,24 @@ module.exports = function() {
   }
 
   /**
+   * Return whether or not an accordion is anchored to in URL
+   *
+   * @param $accordion      jQuery object of accordion element
+   * @returns {boolean}
+     */
+  function isAnchored($accordion) {
+
+    // get accordion's title and convert to lowercase hyphen-separated string
+    var title = $accordion.attr('id');
+
```

Comment:
It makes sense to me. I'm saying that if an accordion is anchored to in the URL, i.e. with a hash `#accordionID`, return true, otherwise return false. What about it could be clearer?